src/Tizen.Network.WiFi/Tizen.Network.WiFi/WiFiNetworkChange.cs

```diff
@@ -294,6 +337,30 @@ private void UnregisterRssiLevelChangedEvent()
             }
         }
 
+        private void RegisterScanStateChangedEvent()
+        {
+            _scanChangedCallback = (int scanState, IntPtr userData) =>
+            {
+                WiFiScanState state = (WiFiScanState)scanState;
+                ScanStateChangedEventArgs e = new ScanStateChangedEventArgs(state);
+                _scanStateChanged.SafeInvoke(null, e);
```

Comment:
How about remove local variable e,state?
And do not use SafeInvoke, please use `?.` operation it is more efficient and more readable
```suggestion
                _scanStateChanged?.(null, new ScanStateChangedEventArgs((WiFiScanState)scanState));
```